packages/server/lib/server.coffee

```diff
@@ -34,6 +36,15 @@ fileServer   = require("./file_server")
 DEFAULT_DOMAIN_NAME    = "localhost"
 fullyQualifiedRe       = /^https?:\/\//
 
+isResponseHtml = (contentType, responseBuffer) ->
+  if contentType
```

Comment:
we probably want to be even more lax than this...

```coffee
if contentType is "text/html"
  return true

## else try to figure out if the body is HTML...
if body = _.invoke(responseBuffer, 'toString')
  return isHtml(body)

return false
```

You can add a test by sending something like `text/plain` as a `content-type` and I believe this will still pass.

Why do it this way? Because it likely better matches what the browser does. Check to see what happens if you send the **wrong** content-type but still serve `html`. My guess is the browser parses and renders the `html` correctly. If I'm wrong avoid this whole comment.

Reply:
I tried sending an HTML body with `text/plain` content-type, Chrome rendered it as plain text:

![image](https://user-images.githubusercontent.com/1151760/58562349-13c39200-81f7-11e9-9726-8b4160e28d59.png)

So this looks like it matches the behavior of Chrome.